packages/harmony/.storybook/preview-body.html

```diff
@@ -10,4 +9,61 @@
   #configuration-options {
     margin-bottom: var(--harmony-spacing-2xl);
   }
+
+  .sbdocs-title {
+    color: var(--text-colors-text, #111);
+    font-size: 72px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+  }
+
+  #storybook-docs :where(h2:not(.sb-anchor, .sb-unstyled, .sb-unstyled h2)) {
+    color: var(--text-colors-text, #111);
+    font-size: 32px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+    border-bottom: unset;
+  }
+
+  #storybook-docs :where(h3:not(.sb-anchor, .sb-unstyled, .sb-unstyled h3)) {
+    color: var(--text-colors-text-muted, #414141);
+    font-size: 24px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: normal;
+  }
+
+  #storybook-docs :where(h4:not(.sb-anchor, .sb-unstyled, .sb-unstyled h4)) {
+    color: var(--text-colors-text-muted, #414141);
+    font-size: 24px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: normal;
+  }
+  
+  #storybook-docs :where(p:not(.sb-anchor, .sb-unstyled, .sb-unstyled p)) {
+    color: var(--text-colors-text-muted, #3F3F3F);
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: normal;
+  }
+
+  #storybook-docs :where(a:not(.sb-anchor, .toc-link, .sb-unstyled, .sb-unstyled a)), li:has(a)::marker{
```

Comment:
this is some css alright 👀

Reply:
indeed :/ fwiw this is based on what the current storybook selectors are, so this is needed to have higher specificity